src/main/java/mate/jdbc/dao/DriverDao.java

```diff
@@ -0,0 +1,17 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Driver;
+
+public interface DriverDao {
+
+    Driver create(Driver driver);
+
+    Driver get(Long id);
```

Comment:
```suggestion
    Optional<Driver> get(Long id);
```

It's better to return Optional from Dao layer